### ISSUE TITLE
MD049 and MD050 do not enforce asterisk style

### ISSUE THREAD

**Issue by TheJaredWilcurt** [DavidAnson/markdownlint#497]
## Dependency versions:

* **markdownlint-cli:** `0.30.0`
  * **markdownlint:** `0.24.0`
    * **markdown-it:** `12.2.0`

## Rules

```yml
# MD049 - Emphasis style should be consistent
emphasis-style:
  style: asterisk

# MD050 - Strong style should be consistent
strong-style:
  style: asterisk
```

## Example

```md
This should _fail_.

So should __this__.
```

**Comment by TheJaredWilcurt** [comment]
If this is already resolved in 0.25.0+, then this issue can be closed with:

* https://github.com/igorshubovych/markdownlint-cli/issues/257

**Comment by DavidAnson** [comment]
Right, the upcoming release of markdownlint-cli will include the version of the library that implements these rules.
